demo/src/views/errors-tooltips-view.js

```diff
@@ -12,107 +12,104 @@ import {
 } from "victory-native";
 import viewStyles from "../styles/view-styles";
 
-export default class extends React.Component {
-  static navigationOptions = {
-    headerTitle: "Errors & Tooltips"
-  };
-
-  render() {
-    return (
-      <ScrollView style={viewStyles.container}>
-        <Text style={[viewStyles.header, viewStyles.monospace]}>
-          {"<VictoryErrorBar />"}
-        </Text>
-
-        <VictoryErrorBar
-          style={{
-            data: { stroke: "red", strokeWidth: 2 }
-          }}
+export default function ErrorsTooltipsView() {
+  return (
+    <ScrollView style={viewStyles.container}>
+      <Text style={[viewStyles.header, viewStyles.monospace]}>
```

Comment:
I think this one should also be nested in `<View pointerEvents="none" >`

Reply:
For this one I can wrap the first chart, but if I do the entire screen then the interactive events for the bottom two won't work. I think when we add the pointer events to primitives it will address this more completely.